hasktorch/src/Torch/NN.hs

```diff
@@ -48,11 +51,36 @@ instance Parameterized Parameter where
   flattenParameters = pure
   replaceOwnParameters _ = nextParameter
 
+instance Parameterized a => Parameterized [a] where
```

Comment:
for this instance maybe you can also use this [generalized version](https://github.com/hasktorch/hasktorch/pull/346/files#diff-1a12fff35c483c6404f1f2749298c9d1) (sorry I hadn't gotten around to fixing the other one there yet, but this one seems good).

Reply:
Thanks for your suggestion.
I've changed the instance with generalized one.
Untyped instance can be fixed with 'OVERLAPS' pragma.
But typed instance can not be fixed with  'OVERLAPS' pragma.(I don't know why.)